### PR TITLE
Build `/lxd` page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Build `/lxd` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [Copy doc](https://docs.google.com/document/d/1LJ6ajHYAmKdxSNDaD5kbriDod1t7RcmIwVJtevAeKFQ/edit#), [Design](https://github.com/canonical-web-and-design/web-squad/issues/4892) 
- Demo at https://ubuntu-com-11378.demos.haus/lxd

## Issue / Card

Fixes [#4893](https://github.com/canonical-web-and-design/web-squad/issues/4893)

## Screenshots
update-lxd

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

**Comment by minkyngkm** [comment]
Thanks @spencerbygraves for reviewing! You are right... I was looking at the design at the top 🤦🏻‍♀️  I will quickly make an update! Thank you!!

**Comment by minkyngkm** [comment]
I've made some updates! Could you please have a look again when you have time? Thanks @spencerbygraves :)

**Comment by spencerbygraves** [comment]
@minkyngkm great, thanks for the updates. If you could just make that image a bit smaller then we're good. I have it 236px wide in my designs.

Thanks.

**Comment by minkyngkm** [comment]
Thanks @spencerbygraves. It's 236 x 214 now!

**Comment by spencerbygraves** [comment]
Thanks!

File: templates/lxd/index.html [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-6">
+      <h1>Run system containers with LXD</h1>
+      <p class="p-heading--4">Fast, dense, and secure container and VM management at any scale </p>
+      <p> LXD provides a unified user experience for managing system containers and virtual machines. For more demanding workloads, LXD can be set up in a cluster environment to run containers, VMs, or a combination of the two on a set of machines. LXD has direct hardware access, minimising overhead and matching the density and efficiency of containers.</p>
+      <p>
+        <a class="p-button--positive" href="https://linuxcontainers.org/lxd/try-it/">Try LXD Online</a>
```
Comment (petesfrench) [inline templates/lxd/index.html:18]: I get a 'Your browser couldn't reach the demo server' message when trying to access the demo

File: templates/lxd/index.html [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+        url="https://assets.ubuntu.com/v1/df54bb6a-document-open-icon.svg",
+        alt="",
+        width="71",
+        height="60",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+      </div>
+      <a href="https://linuxcontainers.org/lxd/docs/master/">Read the documentation </a>
```
Comment (petesfrench) [inline templates/lxd/index.html:226]:
```suggestion
      <a href="https://linuxcontainers.org/lxd/docs/master/">Read the documentation&nbsp;&rsaquo;</a>
```

File: templates/lxd/index.html [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+        url="https://assets.ubuntu.com/v1/3aef5bf9-forum-icon.svg",
+        alt="",
+        width="60",
+        height="60",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+      </div>
+      <a href="https://discuss.linuxcontainers.org/">Discuss with other users</a>
```
Comment (petesfrench) [inline templates/lxd/index.html:240]:
```suggestion
      <a href="https://discuss.linuxcontainers.org/">Discuss with other users&nbsp;&rsaquo;</a>
```

File: templates/lxd/index.html [hunk 4]
```diff
@@ -0,0 +1,350 @@
...
+        url="https://assets.ubuntu.com/v1/3769b4a7-webinar-icon.svg",
+        alt="",
+        width="73",
+        height="60",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+      </div>
+      <a href="https://www.youtube.com/c/LXDvideos/videos">Watch tutorials on YouTube</a>
```
Comment (petesfrench) [inline templates/lxd/index.html:254]:
```suggestion
      <a href="https://www.youtube.com/c/LXDvideos/videos">Watch tutorials on YouTube&nbsp;&rsaquo;</a>
```

File: templates/lxd/index.html [hunk 5]
```diff
@@ -0,0 +1,350 @@
...
+      <p>Run Kubernetes on LXD for edge computing</p>
+      <a class="p-link--inverted u-sv1" href="https://www.brighttalk.com/webcast/6793/470152?utm_source=brighttalk-portal&utm_medium=web&utm_campaign=viewing-history">Watch the webinar&nbsp;&rsaquo;</a>
+    </div>
+  </div>
+</section>
+
+<!-- Set default Marketo information for contact form below-->
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/lxd" data-form-id="3230" data-lp-id="6274" data-return-url="/lxd" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
+</div>
+{% endblock content %}
```
Comment (petesfrench) [inline templates/lxd/index.html:350]: Needs a newline at the end of the file

File: templates/lxd/index.html [hunk 6]
```diff
@@ -0,0 +1,350 @@
...
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-12">
```
Comment (petesfrench) [inline templates/lxd/index.html:138]:
```suggestion
  <div class="u-fixed-width">
```
You can use a `u-fixed-width` to achieve the same thing here, unless you are using it for the every white space that you get from using a `row`

File: templates/lxd/index.html [hunk 7]
```diff
@@ -0,0 +1,350 @@
...
+    </div>
+    <div class="col-4">
+      <h3 class="p-heading--4">Deploy workloads across different platforms with Juju</h3>
+      <p>Juju makes it easy to deploy the exact same software on a variety of different platforms, including public and private cloud, bare metal, Kubernetes clusters. It is also a useful tool for local deployments, ideal for development.</p>
+      <p>For operating your instances using Juju please read <a href="https://juju.is/docs/olm/lxd">this tutorial</a>.</p>
+    </div>
+  </div>
+</section>
+<section class="p-strip--suru-blog-header">
+  <div class="row">
```
Comment (petesfrench) [inline templates/lxd/index.html:283]:
```suggestion
  <div class="u-fixed-width">
```
`row` should only be used with nested `col-`s

File: templates/lxd/index.html [hunk 8]
```diff
@@ -0,0 +1,350 @@
...
+          alt="",
+          width="32",
+          height="28",
+          hi_def=True,
+          loading="lazy",
+          ) | safe
+        }}
+        <h3 class="p-heading-icon__title p-heading--4">Webinar</h3>
+      </div>
+    </div>
```
Comment (petesfrench) [inline templates/lxd/index.html:302]: There is a very small formatting issue here

**Review by petesfrench — CHANGES_REQUESTED** [review]
Looking good, just some minor changes needed

File: templates/lxd/index.html (hunk 1 above)
Comment (minkyngkm) [reply templates/lxd/index.html:18]: I will double check if the link is correct with the PdM. Thanks Pete!

File: templates/lxd/index.html [hunk 9]
```diff
@@ -0,0 +1,349 @@
...
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width u-sv3">
+    <h2 class="u-sv3">Why choose LXD?</h2>
+  </div>
+  <div class="row u-equal-height u-sv3">
+    <div class="col-4">
+      <div class="u-hide--small u-sv3">
```
Comment (sowasred2012) [inline templates/lxd/index.html:42]:
```suggestion
      <div class="u-hide--small u-hide--medium u-sv3">
```

File: templates/lxd/index.html [hunk 10]
```diff
@@ -0,0 +1,349 @@
...
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <h3 class="p-heading--4">Run a full Linux OS inside a container</h3>
+      <p>LXD <a href="/blog/what-are-linux-containers">system containers</a> run a complete filesystem with background processes. This allows you to run any workload, or containerise your traditional systems and apps without modifying the apps or your operations. LXD containers offer the density and efficiency of containers with a VM-like experience.</p>
+    </div>
+    <div class="col-4">
+      <div class="u-hide--small u-sv3">
```
Comment (sowasred2012) [inline templates/lxd/index.html:57]:
```suggestion
      <div class="u-hide--small u-hide--medium u-sv3">
```

File: templates/lxd/index.html [hunk 11]
```diff
@@ -0,0 +1,349 @@
...
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <h3 class="p-heading--4">Cloud-like experience</h3>
+      <p>LXD is image based, providing images for a large number of Linux distributions. It comes with a built-in image store that allows you to spin up a container or a VM in a matter of seconds. Existing integrations with various deployment and orchestration tools allow you to manage your infrastructure in a cloud-like way.</p>
+    </div>
+    <div class="col-4">
+      <div class="u-hide--small u-sv3">
```
Comment (sowasred2012) [inline templates/lxd/index.html:72]:
```suggestion
      <div class="u-hide--small u-hide--medium u-sv3">
```

**Review by sowasred2012 — APPROVED** [review]
LGTM - just a small tweak on the images on medium screens

**Comment by sowasred2012** [comment]
@petesfrench this check is failing because the word "master" is contained in a URL, how do we make it happy?

File: templates/lxd/index.html (hunk 1 above)
Comment (anthonydillon) [reply templates/lxd/index.html:18]: Yes this is the correct link. You should be able to accept the t&cs and create a demo

**Review by petesfrench — APPROVED** [review]
We should be able to ignore it. I think it just needs my review in this case.

**Pull request merged**